x/circuit/keeper/test_utils.go

```diff
@@ -0,0 +1,41 @@
+package keeper
+
+import (
+	"context"
+	"testing"
+
+	dbm "github.com/cosmos/cosmos-db"
+	"github.com/cosmos/cosmos-sdk/store"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/tendermint/libs/log"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	"gotest.tools/v3/assert"
+)
+
+type baseSuite struct {
```

Comment:
Don't we call these fixtures?
```suggestion
type testFixture struct {
```

Reply:
yea can change the name, while reading regens code they called it suites so I went with it. I can change .